test/e2etest/bamboo_test.go

```diff
@@ -278,3 +268,23 @@ func assertBambooAgentPod(t *testing.T, kubectlOptions *k8s.KubectlOptions) {
 		assert.Equal(t, true, pod.Status.ContainerStatuses[0].Ready)
 	}
 }
+
+func assertRestoredDataset(t *testing.T, product string, environment string, domain string) {
+	url := fmt.Sprintf("https://%s.%s.%s/allProjects.action", product, environment, domain)
+	content := getPageContent(t, url)
+	assert.Contains(t, string(content), "<title>All projects - Atlassian Bamboo</title>")
+	assert.Contains(t, string(content), "totalRecords: 1")
```

Comment:
I wanted to verify the name of the project, but the projects are loaded async by javascript:
```
<script>
            require('page/projects', function(ProjectsApplication) {
                new ProjectsApplication({
                    region: '#projectsTable',
                    totalRecords: 1
                }).start();
            })
        </script>
```
because we are using only simple `get` we don't render the page and we have just the HTML.